### ISSUE TITLE
failed to register layer: failed to Lchown

### ISSUE THREAD

**Issue by LordZanna** [intelowlproject/IntelOwl#2390]
## What happened
Launhing "./start prod up --all_analyzers -- -d" I get this error message: 
"failed to register layer: failed to Lchown "/usr/local/lib/node_modules/box-js/node_modules/optionator/CHANGELOG.md" for UID 110779, GID 100 (try increasing the number of subordinate IDs in /etc/subuid and /etc/subgid): lchown /usr/local/lib/node_modules/box-js/node_modules/optionator/CHANGELOG.md: invalid argument"

## Environment
1. OS: Ubuntu 23.04
2. IntelOwl version: 6.0.4
3. Docker version 25.0.2, build 29cf629
4. Docker Compose version v2.24.5

##  What did you expect to happen
I expected it will started wihout errors

## How to reproduce your issue
In sequence i have done:
./initialize.sh
./start prod up -- d
./start prod down
./start prod up --all_analyzers -- -d

## Error messages and logs
"failed to register layer: failed to Lchown "/usr/local/lib/node_modules/box-js/node_modules/optionator/CHANGELOG.md" for UID 110779, GID 100 (try increasing the number of subordinate IDs in /etc/subuid and /etc/subgid): lchown /usr/local/lib/node_modules/box-js/node_modules/optionator/CHANGELOG.md: invalid argument"

**Comment by drosetti** [comment]
It seems a problem related to the user ID range. Did you try to increase it in the files '/etc/subuid` and `/etc/subgid' ?

**Comment by drosetti** [comment]
> Yes, I'm launching docker as root and I have increased to 655350.

Inside the container malware_tools_analyzers ?

**Comment by LordZanna** [comment]
No, I might have made a mistake because I am not familiar with docker. I did it on the host.
I don't know how to modify these files inside the image.

**Comment by drosetti** [comment]
Don't worry, there's no problem :)
In case the container is running you can login with `sudo docker exec -ti intelowl_malware_tools_analyzers /bin/bash` and edit the file `/etc/subuid`
In case it's down I suggest to:
1 - `docker cp intelowl_malware_tools_analyzers:/etc/subuid .` in this way you copy the file from the container to your host
2 - edit the file `subuid`
3 - `docker cp subuid intelowl_malware_tools_analyzers:/etc/subuid`
4 - start again

In this way the file inside the container should be updated

**Comment by LordZanna** [comment]
Thanks for help but I don't have the image because it vanishes after the error mentioned above.
I've tried this:
`docker pull intelowlproject/intelowl_malware_tools_analyzers`

I obtain this message:
"Using default tag: latest
Error response from daemon: manifest for intelowlproject/intelowl_malware_tools_analyzers:latest not found: manifest unknown: manifest unknown"

May I do something else to download it?

**Comment by LordZanna** [comment]
Thanks for help but I don't have the image because it vanishes after the error mentioned above.
Doing `docker images` it doesn't shows up.

I've tried this:
`docker pull intelowlproject/intelowl_malware_tools_analyzers:"v6.0.4"`

I obtain this message:
"failed to register layer: failed to Lchown "/usr/local/lib/node_modules/box-js/node_modules/optionator/CHANGELOG.md" for UID 110779, GID 100 (try increasing the number of subordinate IDs in /etc/subuid and /etc/subgid): lchown /usr/local/lib/node_modules/box-js/node_modules/optionator/CHANGELOG.md: invalid argument"

The image vanished again.

May I do something else to download it?

**Comment by drosetti** [comment]
Sorry if I take time to answer, but I was on vacation.
Talking about the vanishing, it's a bit weird: with `docker images ls -a` I see the image is not in the list.
After I run the command `docker pull intelowlproject/intelowl_malware_tools_analyzers:"v6.0.4"` the image is correctly listed.

Also the error should appear when you run the container, not when you download the image.

**Comment by LordZanna** [comment]
No problem, nothing urgent.
I don't know what I'm doing wrong, but attached you will find what I did and what I can see after doing it.


![2024-06-28_11h00_11](https://github.com/intelowlproject/IntelOwl/assets/32960990/e15d44ab-3230-49c8-bdd4-d055ea53503c)

**Comment by cauliflowerdoughnuts** [comment]
i had the same error trying to run with the --malware_tools_analyzers and the test branch seems to have fixed the issue.

**Comment by mlodic** [comment]
thanks for the feedback. thus I am closing this
